src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0"
+          >
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <main>
+      <section class="product" data-qa="card">
+
+        <img
+          class="product__picture"
+          src="./images/imac.jpeg"
+          alt="iMac picture"
+        >
+
+        <div class="product__info">
+
+          <div class="product__title">
+            <p class="title__name-item">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+            <p class="title__code-item">Код товара: 195434</p>
+          </div>
+
+          <div class="product__feedback">
+            <div class="feedback__rating">
+              <div class="rating__star"></div>
+              <div class="rating__star"></div>
+              <div class="rating__star"></div>
+              <div class="rating__star"></div>
+              <div class="rating__star"></div>
+            </div>
+            <div class="feedback__feedbacks">
+              <a class="feedbacks__count" href="#">Отзывов: 5</a>
+            </div>
+          </div>
+
+          <div class="product__price">
+            <p class="price__label">Цена:</p>
```

Comment:
you don't have price block, do you? 
```suggestion
            <p class="product__price-label">Цена:</p>
```